orttraining/orttraining/training_ops/cuda/optimizer/sg_impl.cu

```diff
@@ -4,7 +4,7 @@
 #include "core/providers/cuda/cuda_common.h"
 #include "core/providers/cuda/cu_inc/common.cuh"
 #include "core/providers/cuda/atomic/common.cuh"
-#include "sg.h"
+#include "sg_impl.h"
```

Comment:
general comment: put the related header first
[https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)